.github/PULL_REQUEST_TEMPLATE/control_plane_template.md

```diff
@@ -1,49 +1,46 @@
+  - I understand I can use [OpenAPI Hub](https://aka.ms/openapihub) to initialize the PR for adding a new version.  
+  - I understand that this doesn't apply if this PR was previously a PR in the [private repository](https://github.com/Azure/azure-rest-api-specs-pr).
+- [ ] I have reviewed following [guidelines](https://aka.ms/rpguidelines), including
+  [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and
+  [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md) (estimated time: 4 hours).  
+  I understand this is required before I can request review from an ARM API Review board.
 
-- [ ] Ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.
+### ARM API changes review
 
-- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them.
+- If you want for the ARM team to review this PR, you must add the `ARMReview` label. 
+- The automation may automatically add the `ARMReview` label, if appropriate.
 
-### Breaking Change Review Checklist
-If you have any breaking changes as defined in the [Breaking Change Policy](https://aka.ms/AzBreakingChangesPolicy/), request approval from the Breaking Change Review Board.
+### Breaking change review
 
-**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Additional details on the process and office hours are on the [Breaking Change Wiki](https://eng.ms/docs/cloud-ai-platform/azure-core/azure-core-pm-and-design/trusted-platform-pm-karimb/service-lifecycle-and-actions-team/service-lifecycle-actions-team/apex/media/overview_breakingchanges).
+If you have any breaking changes as defined in the [Breaking Change Policy](https://aka.ms/AzBreakingChangesPolicy/), 
+follow the process outlined in the [High-level Breaking Change Process doc](https://eng.ms/docs/cloud-ai-platform/azure-core/azure-core-pm-and-design/trusted-platform-pm-karimb/service-lifecycle-and-actions-team/service-lifecycle-actions-team/apex/media/overview_breakingchanges#high-level-breaking-change-process).
+      
+## Getting help
 
-NOTE: To update API(s) in public preview for over 1 year (refer to [Retirement of Previews](http://aka.ms/aprwiki))
-
-Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).
+- For general guidance on the spec PR review process, see https://aka.ms/specprreview and [FAQ](https://aka.ms/faqinprreview).
```

Comment:
>For general guidance on the spec PR review process, see https://aka.ms/specprreview  [](http://example.com/codeflow?start=2&length=84)

Lets promote this to the checklist

Reply:
Consider removing the FAQ link as information there is mostly repeted in the aka.ms/specprreview or it is out of date

Reply:
Thx, I removed the FAQ, and promoted the specpprreview URL to due diligence checklist. Also [I put deprecation notice in it](https://github.com/Azure/azure-rest-api-specs/wiki/FAQ-in-Swagger-PR-Review).